apps/experiments/models.py

```diff
@@ -464,11 +493,8 @@ class Meta:
 
     def save(self, *args, **kwargs):
         if not hasattr(self, "chat"):
-            self.chat = Chat.objects.create(team=self.team, user=self.user, name=self.experiment.name)
```

Comment:
does this mean we can drop the `user` field from the Chat model?

Reply:
Correct. It was ultimately just a duplicate of the experiment session's user, which moved to participant and will always be there. I'll do this in a followup PR